### PR TITLE
Bundle configuration now contains Prefix which control the prefix of …

### PR DESCRIPTION
**Pull request by liorlevtov**
…the path that downloads a bundle.

### CODE REVIEW

File: docs/book/configuration.md [hunk 1]
```diff
@@ -81,6 +82,7 @@ multiple services.
 | Field | Type | Required | Description |
 | --- | --- | --- | --- |
 | `bundle.name` | `string` | Yes | Name of the bundle to download. |
+| `bundle.prefix` | `string` | No | Prefix to add to the name of the bundle |
```
Comment (ashutosh-narkar) [inline docs/book/configuration.md]: `No (default: bundles)` . Also the [Bundle docs](https://www.openpolicyagent.org/docs/bundles.html) need to be updated with the `prefix` explanation similar to the [Discovery docs](https://www.openpolicyagent.org/docs/discovery.html).

File: plugins/bundle/config.go [hunk 2]
```diff
@@ -31,12 +31,13 @@ func ParseConfig(config []byte, services []string) (*Config, error) {
 	return &parsedConfig, nil
 }
 
-// Config represents configuration the plguin.
+// Config represents configuration the plugin.
```
Comment (ashutosh-narkar) [inline plugins/bundle/config.go]: `NIT: Config represents the configuration of the plugin.`

File: plugins/bundle/plugin.go [hunk 3]
```diff
@@ -114,7 +115,7 @@ func (p *Plugin) Unregister(name interface{}) {
 
 func (p *Plugin) initDownloader() {
 	client := p.manager.Client(p.config.Service)
-	path := fmt.Sprintf("/bundles/%v", p.config.Name)
+	path := fmt.Sprintf("/%v/%v", strings.Trim(*(p.config.Prefix), "/"), strings.Trim(p.config.Name, "/"))
```
Comment (ashutosh-narkar) [inline plugins/bundle/plugin.go]: Adding test cases will be useful.

Comment (luminatesec) [reply plugins/bundle/plugin.go]: yep

**Comment by luminatesec** [comment]
done

File: plugins/bundle/config_test.go [hunk 4]
```diff
@@ -24,12 +27,25 @@ func TestConfigValidation(t *testing.T) {
 			input:   `{"name": a/b/c", "service": "service2"}`,
 			wantErr: false,
 		},
+		{
+			input:   `{"name": a/b/c", "service": "service2", "prefix: mybundle"}`,
```
Comment (tsandall) [inline plugins/bundle/config_test.go]:
```suggestion
			input:   `{"name": "a/b/c", "service": "service2", "prefix": "mybundle"}`,
```

Looks like this copied an issue in the test above. Could you correct that one and the one below as well?

Comment (luminatesec) [reply plugins/bundle/config_test.go]: sure, added more tests to catch this scenario.
Done

**Comment by luminatesec** [comment]
done

**Comment by tsandall** [comment]
LGTM. Thanks!

**Pull request merged**